showcase/src/assets/examples/date-picker/reactive-form.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Inject, OnDestroy, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { FormControl } from "@angular/forms";
+import { Subscription } from "rxjs";
+
+@Component({
+	selector: "demo-date-picker",
+	templateUrl: "./demo-date-picker.component.html"
+})
+export class DemoDatePickerComponent implements OnInit, OnDestroy {
+	public formControl: FormControl;
+	public minDate: Date;
+	public maxDate: Date;
+
+	private subscription: Subscription;
+
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService) {}
+
+	public ngOnInit(): void {
+		this.formControl = new FormControl(moment(new Date()));
+		this.minDate = new Date("07-01-2017");
+		this.maxDate = new Date("07-20-2017");
```

Comment:
`this.minDate` and `this.maxDate` are implemented differently in the showcase. That might be confusing.